helm-charts/chatqna/templates/customMetrics.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.horizontalPodAutoscaler.enabled }}
```

Comment:
In addition to overwriting `adapter-config`, does this also restart PrometheusAdapter so that the new custom metrics are avaiable for HPA?

Reply:
No, there is no automatic restarting of arbitrary Pods. Also, we cannot guarantee such a Pod or Deployment existing.

Reply:
Ok, I'll provide blurb for Chart README that instructs user to do that, and tells about the gotchas in enabling HPA.